build/sidecar.Containerfile

```diff
@@ -0,0 +1,25 @@
+# Build the manager binary
```

Comment:
Rename this to `Containerfile.sidecar`, all `Containerfiles` shoul follow this naming convention (will group them together when listing the files in the directory)